Security/src/CloudFoundrySingleSignon/Controllers/HomeController.cs

```diff
@@ -16,32 +16,34 @@ public class HomeController : Controller
     {
         private readonly HttpClient httpClient;
         private readonly ILogger<HomeController> _logger;
+        private readonly IHttpContextAccessor _httpContextAccessor;
 
-        public HomeController(IHttpClientFactory clientFactory, ILogger<HomeController> logger)
+        public HomeController(IHttpContextAccessor httpContextAccessor, IHttpClientFactory clientFactory, ILogger<HomeController> logger)
         {
+            _httpContextAccessor = httpContextAccessor;
```

Comment:
This field is never used. Can it be removed (including the constructor parameter)?